ydb/core/protos/local.proto

```diff
@@ -53,6 +63,7 @@ message TEvStatus {
     optional uint64 AvailableWeight = 5;
     optional NKikimrTabletBase.TMetrics ResourceMaximum = 8;
     optional uint64 StartTime = 7;
+    optional TActorSystemInfo ActorSystemInfo = 9;
```

Comment:
TEvStatus is sent when Local first registers in Hive, and when the tenant is altered. If we want this info to be sent regularly, I would put in TabletMetrics message.